migration/v0.36/apply/11-kube-prometheus-stack.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "  - Replace kube-prometheus-stack CRDs on sc"
+      kubectl_do sc replace -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+
+      log_info "  - Upgrade kube-prometheus-stack on sc"
+      helm_do sc upgrade kube-prometheus-stack  prometheus-community/kube-prometheus-stack -n monitoring -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/values.yaml
```

Comment:
any reason for not using `helmfile_upgrade sc app=chart-label`?

Reply:
Made sense!